src/NativeShell/src/media/RemotePlayerService.java

```diff
@@ -17,28 +17,48 @@
 import android.media.session.PlaybackState;
 import android.os.Build;
 import android.os.IBinder;
-import android.support.v4.media.session.MediaSessionCompat;
-import android.util.Log;
 
+import com.mb.android.api.ApiClientBridge;
 import com.mb.android.logging.AppLogger;
 
 import org.jellyfin.mobile.MainActivity;
 import org.jellyfin.mobile.R;
+import org.jellyfin.cordova.NativeShell;
 
 import mediabrowser.apiinteraction.Response;
 import mediabrowser.apiinteraction.android.VolleyHttpClient;
 import mediabrowser.apiinteraction.android.mediabrowser.Constants;
 import mediabrowser.model.logging.ILogger;
 
 public class RemotePlayerService extends Service {
-
+    private ApiClientBridge apiClientBridge;
     private VolleyHttpClient httpClient;
     private MediaController mediaController;
     private MediaSessionManager mediaSessionManager;
     private MediaSession mediaSession;
-    private String mediaSessionMediaId = "";
     private Bitmap largeItemIcon;
 
+    private String mediaSessionId;
+    private String channelId;
+    private int notifyId = 84;
+
+    @Override
+    public void onCreate() {
+        apiClientBridge = new ApiClientBridge(getApplicationContext());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+
+            channelId = "Channel";
```

Comment:
huh? what's this string used for?

Reply:
Can we make it more specific, like, I dunno, `JellyfinChannel`? To avoid possible clashes or something like that...